app/components/certificate/certificate-available.tsx

```diff
@@ -28,14 +28,14 @@ export default function CertificateAvailable({
       <CertificateDisplay
         title="Public Key"
         value={publicKey}
-        description="Public: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s"
+        description="Public key is intended to be shared widely and is used for encrypting data. Public key is used to encrypt data that can only be decrypted by the corresponding private key. In other words, anyone can use the public key to encrypt data, but only the owner of the private key can decrypt it."
         pathname="cert"
       />
       <Divider />
       <CertificateDisplay
         title="Private Key"
         value={privateKey}
-        description="Private: Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s"
+        description="Private key is a secret and is used for decrypting data that has been encrypted using the corresponding public key. Private key is used for creating digital signatures to verify the authenticity of digital documents, messages, and transactions."
```

Comment:
```suggestion
        description="Private key is a secret and is used for decrypting data that has been encrypted using the corresponding public key."
```
Do we need the second sentence?

Reply:
I agree with your suggestion!